SayTheirNames/Source/View/HashtagView.swift

```diff
@@ -34,14 +34,15 @@ class HashtagView: UIView {
     
     // MARK: - Configure Subview
     private func configureView() {
+        let mediumPadding = Theme.Components.Padding.medium
         layer.borderWidth = 1.5
         layer.borderColor = UIColor(asset: STNAsset.Color.primaryLabel).cgColor
         hashtagLabel.anchor(superView: self,
                             top: topAnchor,
                             leading: leadingAnchor,
                             bottom: bottomAnchor,
                             trailing: trailingAnchor,
-                            padding: UIEdgeInsets(top: 5, left: 5, bottom: 5, right: 5), size: .zero)
+                            padding: UIEdgeInsets(top: 5, left: mediumPadding, bottom: 5, right: mediumPadding), size: .zero)
```

Comment:
I think the hashtags look better in the "before" version - more proportional. So here it'd use `Padding.tiny` instead of `.medium`

Reply:
Oops, didn't see this comment before I approved this PR, sorry.

My 2 cents, I kinda like the look of the new hashtags, they have a bit more room to breathe.

But don't want it to be a blocker, whatever we go is fine with me.

Reply:
Thanks for the feedback, I'll change it to use Padding.tiny 🙂.